### PR TITLE
Add notify

### PR DESCRIPTION
**Pull request by andreychuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+
 Postgresql:
 
 ```
 bash-3.2$ docker-compose exec postgresql psql -U postgres
 psql (9.5.3)
 Type "help" for help.
 
-postgres=# postgres=# SELECT pg_notify('mychannel', 'Ololo!');
+postgres=# postgres=# SELECT pg_notify('messanger', 'Ololo!');
```
Comment (wildsurfer) [inline README.md]: there is typo here. messEnger

Comment (andreychuk) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
 Readme coming soon...
 ---------------------
 
+For correct work you need to run npm install in `notify/app`
+```
+cd notify/app
+npm install
```
Comment (wildsurfer) [inline README.md:7]: Is it possible to do this in Dockerfile?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+consumer/src/vendor/*
```
Comment (wildsurfer) [inline .gitignore:1]: Do we really need `.idea` folder for this example? Let's remove it to keep it simple

File: docker-compose.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ services:
   postgresql:
     build: ./postgresql
     ports:
-      - "5432"
+      - "5432:5432"
```
Comment (wildsurfer) [inline docker-compose.yml:6]: It is not secure to open port 5432 to the world

File: docker-compose.yml [hunk 5]
```diff
@@ -22,10 +22,21 @@ services:
     mem_limit: 256m
     ports:
       - "5672"
-      - "15672"
+      - "15672:15672"
   consumer:
     build: ./consumer
     depends_on:
       - rabbitmq
     restart: always
```
Comment (wildsurfer) [inline docker-compose.yml:30]: Let's remove `restart: always` cause it isn't essential for this example

File: docker-compose.yml [hunk 6]
```diff
@@ -22,10 +22,21 @@ services:
...
+    environment:
+      - POSTGRES_PASSWORD=DGH5k39xchgh3Po
+      - POSTGRES_USER=postgres
+      - POSTGRES_DATABASE=test
+      - POSTGRES_HOST=postgresql
+      - RABBIT_CONNECTION_STRING=amqp://guest:guest@rabbitmq
+    depends_on:
+      - postgresql
+      - rabbitmq
+    restart: always
```
Comment (wildsurfer) [inline docker-compose.yml:42]: Let's remove `restart: always` cause it isn't essential for this example

File: notify/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,10 @@
+FROM node:6.9.2-alpine
+COPY ./app/config.json /root/
+COPY ./app/ /var/www/html/web/
+COPY ./src/wait /root/
+RUN chmod +x /root/start.sh
+RUN chmod +x /root/run.sh
```
Comment (wildsurfer) [inline notify/Dockerfile:6]: Doing `chmod` inside of Dockerfile looks wired. Instead I recommend to chmod files in git repository

File: notify/src/default.conf [hunk 8]
```diff
@@ -0,0 +1,22 @@
+server {
```
Comment (wildsurfer) [inline notify/src/default.conf:1]: Do we need nginx config here?!

File: notify/src/wait/wait-postgres.sh [hunk 9]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```
Comment (wildsurfer) [inline notify/src/wait/wait-postgres.sh:1]: Do we use this script at all?

File: notify/src/supervisor/README.md [hunk 10]
```diff
@@ -0,0 +1,2 @@
+https://www.digitalocean.com/community/tutorials/how-to-install-and-manage-supervisor-on-ubuntu-and-debian-vps
```
Comment (wildsurfer) [inline notify/src/supervisor/README.md:1]: Do we need this README?

File: notify/src/wait/wfi/LICENSE [hunk 11]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```
Comment (wildsurfer) [inline notify/src/wait/wfi/LICENSE:1]: `wait/wfi` seems to be some package. is it possible to add it as a package? Storing it here looks wired

**Review by wildsurfer — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (andreychuk) [reply README.md:7]: done

File: docker-compose.yml (hunk 4 above)
Comment (andreychuk) [reply docker-compose.yml:6]: done

File: docker-compose.yml (hunk 5 above)
Comment (andreychuk) [reply docker-compose.yml:30]: done

File: docker-compose.yml (hunk 6 above)
Comment (andreychuk) [reply docker-compose.yml:42]: done

File: notify/Dockerfile (hunk 7 above)
Comment (andreychuk) [reply notify/Dockerfile:6]: done

File: notify/src/wait/wait-postgres.sh (hunk 9 above)
Comment (andreychuk) [reply notify/src/wait/wait-postgres.sh:1]: yep

File: notify/src/supervisor/README.md (hunk 10 above)
Comment (andreychuk) [reply notify/src/supervisor/README.md:1]: nope, removed

File: notify/src/wait/wfi/LICENSE (hunk 11 above)
Comment (andreychuk) [reply notify/src/wait/wfi/LICENSE:1]: moved to Dockerfile

**Pull request closed without merging**